### PR TITLE
Update closed-loop-control.md

### PR DESCRIPTION
**Pull request by brettle**
Document the PID loop frequency and the units of the output, setpoint, and the various constants.

### CODE REVIEW

File: operating-modes/closed-loop-control.md [hunk 1]
```diff
@@ -9,7 +9,15 @@ Below is a diagram and the firmware implementation of the internal SPARK MAX PID
 ![](https://cdn11.bigcommerce.com/s-t3eo8vwp22/product_images/uploaded_images/pidfdiagram2.png)
 
 ```c
-//Synchronous PID, call at desired frequency
+// Synchronous PID, called at 1kHz.
+// The output is always in normalized units (i.e. 1 = full forward, -1 = full reverse).
+// setpoint is always in native units for the corresponding control method (e.g. rotations, RPM, Amps, Volts).
```
Comment (auscompgeek) [inline operating-modes/closed-loop-control.md:14]: Should this mention the configured conversion factors will affect the setpoint units?

Comment (brettle) [reply operating-modes/closed-loop-control.md]: I've tried to address that in my latest commit. Let me know if that helps.

Comment (brettle) [reply operating-modes/closed-loop-control.md]: Does my latest commit address this sufficiently?

**Comment by brettle** [comment]
Ping. Any reason this hasn't been merged?

**Pull request merged**